integr8ly/keycloak-operator
# feat: Update sso plugins init image version

## Motivation
Additional updates have been made to the keycloak-metrics-spi - https://github.com/aerogear/keycloak-metrics-spi/releases. Updating the version of the sso plugins init image which contains the keycloak-metrics-spi

## Verification Steps
1. NAvigate to https://sso-sso.apps.dffrench-30ed.openshiftworkshop.com/auth/realms/master/metrics
3. Verify you can see the metric `keycloak_request_duration`

## Checklist:


## Progress


## Additional Notes
